libraries/Microsoft.Bot.Builder.AI.LUIS/LuisRecognizerOptionsV3.cs

```diff
@@ -164,12 +164,15 @@ private async Task<RecognizerResult> RecognizeAsync(ITurnContext turnContext, st
             {
                 var uri = BuildUri(options);
                 var content = BuildRequestBody(utterance, options);
-                httpClient.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", Application.EndpointKey);
-                using (var stringContent = new StringContent(content.ToString(), System.Text.Encoding.UTF8, "application/json"))
+
+                using (var request = new HttpRequestMessage(HttpMethod.Post, uri.Uri))
                 {
-                    var response = await httpClient.PostAsync(uri.Uri, stringContent, cancellationToken).ConfigureAwait(false);
+                    request.Content = new StringContent(content.ToString(), Encoding.UTF8, "application/json");
```

Comment:
Please add a `using` for the StringContent (since it is disposable)

Reply:
Done 👍